scripts/get_version.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import argparse
+from typing import Dict
+
+import requests
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("version_type", choices=["stable", "latest", "current"])
+    return parser.parse_args()
+
+
+def get_current_version() -> str:
+    VERSION: Dict[str, str] = {}
+    with open("allennlp_models/version.py", "r") as version_file:
+        exec(version_file.read(), VERSION)
```

Comment:
Couldn't you just import that file?

Reply:
Well yes hahah, I copied this from the core repo without thinking much. But actually I do think it's beneficial to keep it like this because
(A) it's faster than importing
(B) in some cases we might want to run this script without actually installing `allennlp_models` or any of its dependencies